models/waypoint_model.go

```diff
@@ -9,7 +9,7 @@ const (
 	Launch   Designation = "launch"
 	Land     Designation = "land"
 	Obstacle Designation = "obstacle"
-	Payload  Designation = "payload"
+	// Payload  Designation = "payload"
```

Comment:
Is there a reason why this is commented?

Reply:
When it is uncommented, I get this error:

`models/waypoint_model.go:12:2: Payload redeclared in this block models/payload_model.go:3:6: other declaration of Payload`

Which seems to go away when the line is commented out

Reply:
The `Payload` enum doesn't specifically have a Designation namespace attached to it, so it conflicts with `type Payload`, consider renaming to something like `PayloadDesignation`?